corehq/apps/hqwebapp/static/hqwebapp/scss/data_cleaning.scss

```diff
@@ -0,0 +1,88 @@
+@import "functions";
+@import "commcarehq/variables";
+@import "variables";
+@import "variables-dark";
+
+.dc-diff-view {
+  font-family: $font-family-monospace;
+  letter-spacing: 1px;
+}
+
+.dc-diff-before,
+.dc-diff-after {
+  border: 1px solid white;
+  margin: $spacer *.13;
+  padding-right: $spacer *.5;
+  vertical-align: middle;
+
+  &::before {
+    display: inline-block;
+    color: $white;
+    line-height: 12px;
+    padding: $spacer *.25 $spacer *.5;
```

Comment:
Nitpick: leading zeroes and consistency with spaces. Sounds like prettier can address this on its own

Reply:
https://github.com/dimagi/commcare-hq/pull/36245